stylebot/editor/style.js

```diff
@@ -616,13 +623,20 @@ stylebot.style = {
    * @param {String} css The CSS to apply to the page.
    */
   resetPreview: function() {
+    // needsReset: only reset stuff that really needs a reset. This is to prevent the resetPreview() fired from port.onDisconnect() to override the enable/disable style toggle and the PreviewPopover
+    if (this.needsReset) {
+      this.needsReset = false;
+
+      if (this.status) {
```

Comment:
Shouldn't you indent the rest of the block?

Reply:
indeed